.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,6 @@
 ---
+name: Standard issue template
+about: A standard template to follow when creating an issue in this repository
```

Comment:
Note: These fields may show up if we ever decide to create multiple issue templates for this repo, so feel free to suggest different text here if you think it can be improved :)